editor/src/core/shared/element-path-tree.ts

```diff
@@ -133,6 +134,73 @@ export function reorderTree(trees: ElementPathTrees, metadata: ElementInstanceMe
   }
 }
 
+function maybeReorderDynamicChildren(
+  children: ElementPathTree[],
+  metadata: ElementInstanceMetadataMap,
+): ElementPathTree[] {
+  function keepOnlyUnique(path: ElementPath, index: number, array: ElementPath[]) {
+    return array.findIndex((e) => EP.pathsEqual(path, e)) === index
```

Comment:
Even better idea: you can just `Utils.uniqBy`

Reply:
We really need to get rid of `Utils`. We separated all of the important parts into separate utils files, so the function you're talking about is actually in `array-utils.ts`: https://github.com/concrete-utopia/utopia/blob/c64919e1e2de7856b801dd8ba083ca48acd7d6d4/editor/src/core/shared/array-utils.ts#L149